### DISCUSSION TITLE
How to use the MicroTeX wasm in javascript?

### DISCUSSION THREAD

**Discussion by AkagiYui** [NanoMichael/MicroTeX#184] [Q&A] [answered]
Is there any uncompressed `index.js` source code and exported function documentation of `microtex-wasm.wasm` in the `microtex-page` branch?

**Comment by sp1ritCS** [comment, marked as answer]
The wasm platform is located here: https://github.com/NanoMichael/MicroTeX/tree/openmath/platform/wasm

**Reply by AkagiYui** [reply]
Thank you, I understand. This is very helpful to me.
